unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Textures/DCLTexture.cs

```diff
@@ -121,7 +121,12 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                     {
                         texture.wrapMode = unityWrap;
                         texture.filterMode = unitySamplingMode;
-                        texture.Compress(false);
+
+                        if (!DataStore.i.performance.disableTextureCompression.Get())
+                        {                       
+                            texture.Compress(false);
+                        }
```

Comment:
shouldn't we use preprocessor directive `#if UNITY_EDITOR` here and everywhere else in this PR?

Reply:
Yes and no, I don't want to lose the ability to compress textures in editor since we need to run the game as close as possible with production settings. Hence the toggle

Reply:
what about
```
#if UNITY_EDITOR
                        if (!DataStore.i.performance.disableTextureCompression.Get())
                        {                       
                            texture.Compress(false);
                        }
#else
                        texture.Compress(false);
#endif               
```         
so the check if avoided in prod

Reply:
Yeah I can live with that